### DISCUSSION TITLE
The master branch being the master branch

### DISCUSSION THREAD

**Discussion by stappersg** [hecrj/iced#1982] [General]
Commits like
* 529d5110cb Merge branch 'master' into 0.10
* 18584a925a Merge branch 'master' into 0.9
* 0e1a9e653f Merge branch 'master' into 0.8
indicate merging into the wrong direction.

![image](https://github.com/iced-rs/iced/assets/35434094/4879e5ec-4d82-4254-a66e-c85cbdcf4f74)

_N commits ahead of master_ is something for a feature branch, not for something that will be become a version.

What are the opinions of letting the master branch be the master branch?

**Comment by hecrj** [comment]
No such thing as merging in the wrong direction.

This is not a discussion worth having.

**Reply by stappersg** [reply]
Suggestion for creating _future release branches_, be in `master` branch and do `git branch -c 0.11`.

Later on, when a next release is due, `git branch -c 0.12`.

**Reply by hecrj** [reply]
Our release branches are different from `master` on purpose. There are certain changes in `master` that simply can't be released yet.

Suggestion for _future_ comments: don't assume you know more about a project than the people involved with it. Don't assume they don't know basic stuff either.

Thanks for teaching me how to create a Git branch. I honestly don't know how I made it this far without such knowledge.
